xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_types.h

```diff
@@ -527,15 +527,20 @@ extern "C" {
 
   } ATTRIBUTE_PACKED PVR_TIMER;
 
+  /*!
+   * @brief special PVR_RECORDING.iSeriesNumber and PVR_RECORDING.iEpisodeNumber value to indicate it is not to be used
+   */
+  const unsigned int PVR_RECORDING_INVALID_SERIES_EPISODE = EPG_TAG_INVALID_SERIES_EPISODE; /*!< @brief recording has no valid season and/or episode info. */
```

Comment:
About them with docs becomes updated on my request, but it make not directly sense about doxygen to have description before and behind.

Further where also a bug on the other places, does doxygen not use a `/*!<` a with `@brief` defined text. Only without this `@brief`

Maybe better to make directly the doc together:
```cpp
  /*!
   * @brief special PVR_RECORDING.iSeriesNumber and PVR_RECORDING.iEpisodeNumber
   *  value to indicate it is not to be used
   *
   *  Used if recording has no valid season and/or episode info.
   */
  const unsigned int PVR_RECORDING_INVALID_SERIES_EPISODE = EPG_TAG_INVALID_SERIES_EPISODE;```

Reply:
Ok, cool. Done ;)

Reply:
There was on begin wrongly by c&p two characters on `*  Used if` see edited text.

Reply:
Ok, fixed now ;)